true-index.html

```diff
@@ -7,7 +7,10 @@
 <div class="nav-hero-container">
     <div class="hero">
         <div class="container">
+            <h1 class="hero-title">Istio</h1>
+            <img class="hero-logo" src="{{home}}/img/istio-logo.svg" />
```

Comment:
- ./_site/true-index.html
  *  image /img/istio-logo.svg does not have an alt attribute (line 1)